### PR TITLE
Changed to PlayerPedId instead of GetPlayerPed(-1)

### PR DESCRIPTION
**Pull request by bandidom9**
Better and faster to use PlayerPedId instead of GetPlayerPed(-1)

### CODE REVIEW

File: web/basic/footer.html [hunk 1]
```diff
@@ -5,13 +5,12 @@
...
-                    <a href="https://github.com/tabarra" class="credit-links" target="_blank">Tabarra</a> for the
-                    <a href="https://github.com/citizenfx" class="credit-links" target="_blank">CitizenFX
-                        Collective</a>.
+                <div class="mx-auto d-md-none"> v{{it.txAdminVersion}} atop FXServer {{it.fxServerVersion}} plus Bandi Mod v0.0.69</div>
+                <div class="d-sm-down-none"> Modificado por Bandido para Puerto Rico Island RP</div>
+                <div class="ml-auto d-sm-down-none"> v{{it.txAdminVersion}} atop FXServer {{it.fxServerVersion}} plus Bandi Mod v0.0.69</div>
+            </footer> 
                 </div>
-                <div class="ml-auto d-sm-down-none"> <strong>v{{it.txAdminVersion}}</strong> atop FXServer <strong>{{it.fxServerVersion}}</strong></div>
+                <div class="ml-auto d-sm-down-none"> <strong>v{{it.txAdminVersion}}</strong> atop FXServer <strong>{{it.fxServerVersion}}</strong> plus Bandi Mod <strong>v0.0.69</strong> </div>
```
Comment (lze3) [inline web/basic/footer.html:13]: Uh? Discard local changes, perhaps? Also, why change this in the first place?

File: web/basic/footer.html [hunk 2]
```diff
@@ -105,14 +104,20 @@ <h5>Ban Player:</h5>
...
+											<option value="12 hours">12 hours</option>
+											<option value="1 day">1 day</option>
+											<option value="2 days">2 days</option>
+											<option value="3 days">3 days</option>
+											<option value="5 days">5 days</option>
+											<option value="1 week">1 week</option>
+											<option value="2 weeks">2 weeks</option>
+											<option value="1 months">1 month</option>
+											<! --<option value="permanent" class="font-weight-bold text-danger" selected>permanent</option>-->
+											<! --<option value="custom" class="font-weight-bold">custom</option> -->
```
Comment (lze3) [inline web/basic/footer.html:120]: Breaking indent and commenting out _features_? Again, keep local changes to yourself.

**Review by lze3 — CHANGES_REQUESTED** [review]

**Comment by lze3** [comment]
Just rebase with `-i` to have a clean tree.

**Comment by lze3** [comment]
Hello? You're committing endlessly your local changes which have no relevance to the PR, please stop.

**Comment by tabarra** [comment]
**the fuck is that?**  
You even removed txAdmin logo...  
  
You basically violated our copyright and license.  
And considering we are an open source project, with a quite permissive license, then that's quite literally a **cunt** move.

**Pull request closed without merging**